plugins/oauth/components/AuthenticatedUserAccountDropdown.js

```diff
@@ -5,46 +5,21 @@ import UserAccountDropdown from "#SRC/js/components/UserAccountDropdown";
 import UserAccountDropdownTrigger from "#SRC/js/components/UserAccountDropdownTrigger";
 
 class AuthenticatedUserAccountDropdown extends UserAccountDropdown {
-  getUserLabel() {
-    const user = AuthStore.getUser();
+  constructor() {
+    super(...arguments);
 
-    let userLabel = null;
-
-    if (user && !user.is_remote) {
-      userLabel = user.description;
-    } else if (user && user.is_remote) {
-      userLabel = user.uid;
-    }
+    this.userLabel = AuthStore.getUserLabel();
+  }
 
-    return userLabel;
+  getTrigger() {
+    return <UserAccountDropdownTrigger content={this.userLabel} />;
```

Comment:
we could just use `AuthStore.getUserLabel()` and get rid of constructor.

Reply:
Then, I'd have to call `AuthStore.getUserLabel()` twice. Is that preferable to having a constructor?

Reply:
We used to call the logic twice, no strong opinion on that. Please use your best judgement

Reply:
In that case, I will leave as is.